rosa_getting_started/rosa-quickstart-guide-ui.adoc

```diff
@@ -1,7 +1,9 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="rosa-quickstart-guide-ui"]
 = {product-title} quick start guide
+
 include::_attributes/attributes-openshift-dedicated.adoc[]
+
```

Comment:
Adding spaces to the headers _removes_ the topic map. These line breaks should be removed